src/blocktypes/installables.nix

```diff
@@ -22,47 +22,56 @@
       flake,
       fragment,
       fragmentRelPath,
+      target,
     }: [
-      (import ./actions/build.nix flake fragment)
+      (import ./actions/build.nix target)
+      # profile commands require a flake ref
       {
         name = "install";
         description = "install this target";
         command = ''
+          # ${target}
           nix profile install ${flake}#${fragment}
```

Comment:
We still need to abolish `${flake}`, here and consequentially in the interface.

Reply:
perhaps we may be able to find some way to forcibly reattach the git information to this `self` somehow? It would be useful for the commands that require a flake ref at least.

Reply:
The quick solution is to consume `PRJ_ROOT` on the checkout.

That allows us to deprecate immediately.

Reply:
you mean set `PRJ_ROOT` manually in std-action? That should be fairly simple using `${{ github.workspace }}`

Reply:
Good idea! :+1:

Reply:
And then, we can replace all uses of `${flake}` with `$PRJ_ROOT` until these issues are fixed upstream.

We can also double use this patterns for actions that require a writable source.

Reply:
implemented in 26c51e2e90b9b93d03582aca135409295a27814e